src/awkward_pandas/array.py

```diff
@@ -108,6 +108,14 @@ def _binop(self, other):
     def _reduce(self, name: str, *, skipna: bool = True, axis=None, **kwargs):
         return getattr(ak, name)(self._data, **kwargs)
 
+    def _explode(self):
+        nums = ak.num(self._data, axis=1)
+        nums_filled = ak.fill_none(nums, 0)
+        data = ak.where(nums_filled == 0, [[None]], self._data)
+        flat = ak.flatten(data)
+        arr = type(self)(flat)
+        return arr, ak.num(data, axis=1)
```

Comment:
When done on a series, what's the return type of explode()? Just curious.

Reply:
Just a `Series` again. The two returns here are consumed by `Series.explode`, it uses the second return to build a new `Index` object and then passes the array and the index objects to `Series._constructor`:

https://github.com/pandas-dev/pandas/pull/54834/files#diff-a5257444a1b322d619680fc77361cc6ea11ef36b363b4bb2289fdef0f41feb70R4403